src/lambdas/lambda_update_file.py

```diff
@@ -0,0 +1,78 @@
+import base64
+import io
+from typing import Dict
+from .common import *
+
+
+def build_expression(data: Dict):
+    expression_attr_names = {}
+    expression_attr_vals = {}
+    for i, (k, v) in enumerate(data.items()):
+        expression_attr_names[f'#f{i}'] = k
+        expression_attr_vals[f':f{i}'] = v
+    update_expression = "SET "
+    for i in range(len(expression_attr_vals)):
+        update_expression += f'#f{i} = :f{i}, '
+    update_expression = update_expression[:-2]
+
+    return update_expression, expression_attr_names, expression_attr_vals
+
+
+def lambda_update_file(event: Dict, context):
+    body: Dict = json.loads(event['body'])
+    headers: Dict = event['headers']
+
+    username: str = jwt_decode(headers)
+    if not user_exists(username):
+        return http_response("Forbidden", 401)
+    
+    metadata: Dict = body['metadata']
+
+    primary_key = {
+        CONTENT_METADATA_TB_PK: metadata['username'],
+        CONTENT_METADATA_TB_SK: metadata['uuid']
+    }
+
+    # Get the whole object
+
+    response = dynamo_cli.get_item(
+        TableName=CONTENT_METADATA_TB_NAME,
+        Key=python_obj_to_dynamo_obj(primary_key)
+    )
+
+    if 'Item' not in response:
+        return http_response(f"No element under key {metadata['old_name']} exists!", 404)
+    
+    # Get all fields that were updated (except key fields!)
+
+    new_obj: Dict = {}
+    for k, v in metadata.items():
+        if k not in ['username', 'uuid']:
```

Comment:
Use `CONTENT_METADATA_TB_{PK, SK}` instead of `{username, uuid}`. There are a few other places where we use magic literals, sort those out as well

Reply:
[a812308](https://github.com/magley/klau-drive/pull/24/commits/a8123087cb0c086a9024e969c1475649a081d578)